spec/javascript/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -44,6 +44,14 @@ describe('document-capture/components/document-capture', () => {
     window.location.hash = originalHash;
   });
 
+  it('does not render the selfie capture by default', () => {
```

Comment:
Is there a plan to add a test for rendering the selfie capture when the feature flag is on?

Reply:
Yep! Still open to feedback, but my plan is basically:
- With this PR I added the minimum tests to make sure this feature doesn't end up in front of users inadvertently.
- With a followup ticket/PR I'll add a LOT of tests. Some of which sill do what you're asking, test with and without the feature flag.

Reply:
Sounds good to me!